discovery-provider/ddl/migrations/0025_add_track_price_history_fkey.sql

```diff
@@ -1,4 +1,12 @@
 -- add this fkey constraint to cascade delete track_price_history on revert
 begin;
-alter table track_price_history add constraint track_price_history_blocknumber_fkey foreign key (blocknumber) references blocks (number) on delete cascade;
+
```

Comment:
This was broken and blocking future migrations from running.
fyi @isaacsolo

Reply:
on prod? all nodes have already completed this migration

Reply:
it's not idempotent and you can't do alter table add constraint if not exists, so it has to be in an if block

Reply:
yea all nodes migrated this in august so idempotency shouldn't be important unless you change the file like w this change it'll run but be a no-op.

not sure how it could break tests if CI test-discovery-provider is passing, maybe just locally

Reply:
@isaacsolo Would it be an issue for new nodes bootstrapping?

Reply:
ok yeah i think this is actually just causing an error msg vs breaking...  @isaacsolo ok if i merge?

Reply:
it wouldn't be an issue for new nodes and yea cool to merge it'll no-op